cdk/lib/new-product-api.ts

```diff
@@ -97,14 +95,13 @@ export class NewProductApi extends GuStack {
 		});
 
 		// ---- Alarms ---- //
-		new GuAlarm(this, 'ApiGateway4XXAlarm', {
+		new SrLambdaAlarm(this, 'ApiGateway4XXAlarm', {
 			app,
 			alarmName: `new-product-api-${this.stage} API gateway 4XX response`,
 			alarmDescription: 'New Product API received an invalid request',
 			evaluationPeriods: 1,
 			threshold: 6,
-			actionsEnabled: isProd,
-			snsTopicName: alarmTopic,
+			lambdaFunctionNames: addSubscriptionLambda.functionName,
```

Comment:
Should this include both function names?
```suggestion
			lambdaFunctionNames: [
				addSubscriptionLambda.functionName,
				productCatalogLambda.functionName,
			],
```